lib/auth_web/templates/app/show.html.eex

```diff
@@ -15,15 +15,15 @@
       <div style="font-size: 1.2em;
       overflow-wrap: break-word; font-family: monospace;
       border: 1px solid black; padding: 10px; background-color: #EBEDEF;">
-      <%= apikey.client_id %>/<%= apikey.client_secret %>
+      <%= apikey.client_id %>/<%= apikey.client_secret %>/<%= @auth_url %>
```

Comment:
While reading this code I've realised that this will work fine for our two instances of the `auth` app. (_so fine for now_) but at some point we might want to store the `auth_url` in the DB so that if we ever need to migrate the DB or instance to a new URL we can still verify that a request is valid. 💭